src/components/Slider/Slider.js

```diff
@@ -0,0 +1,26 @@
+import { Swiper, SwiperSlide } from "swiper/react";
```

Comment:
Although is not necessary, try to not use libraries in the project. For instance here you could make a swiper manually or alternatively see how this library works under the hood and try to replicate it

Reply:
Thanks for the comment!